src/main/resources/screens/live/IEDET_SEND_INPUT.json

```diff
@@ -22,18 +24,40 @@
       },
       "component": "entity-list",
       "name": "FinalResponseDocument",
-      "label": "Upload the Final Response"
+      "label": "Upload the final response"
     },
     {
       "validation": [
         "required"
       ],
       "props": {
-        "choices": "S_IEDET_CASE_OUTCOME"
+        "choices": [
```

Comment:
I think this value also appears in the accordion and summary, so these choices will also need updating in the configs for those in casework.